packages/dds/tree/src/test/tree/visitDelta.spec.ts

```diff
@@ -81,9 +87,17 @@ function testDeltaVisit(
 	let callIndex = 0;
 	const result: VisitScript = [];
 	const makeChecker =
-		(name: string) =>
+		(name: keyof DeltaVisitor) =>
 		(...args: unknown[]) => {
-			result.push([name, ...args] as VisitCall);
+			const call: VisitCall =
+				name === "create"
+					? ([
+							name,
+							(args[0] as readonly ITreeCursorSynchronous[]).map(mapTreeFromCursor),
```

Comment:
I think this is assuming its safe because it matches the parameters type from the create method? It would be nice if this code was written such that it would get a type error if said method had its types changed. I think casting args to something like Paramaters<typeof DeltaVisitor.create> would be more clear and more safe.